utilities/infra.py

```
@@ -192,12 +194,20 @@ def get_pod_container_error_status(pod: Pod) -> str | None:
+def _is_console_plugin_bug_open() -> bool:
+    return is_jira_open(jira_id="CNV-82451")
+
+
 def get_not_running_pods(pods: list[Pod], filter_pods_by_name: str = "") -> list[dict[str | None, str]]:
     pods_not_running = []
     for pod in pods:
         if filter_pods_by_name and filter_pods_by_name in pod.name:  # type: ignore[operator]
             LOGGER.warning(f"Ignoring pod: {pod.name} for pod state validations.")
             continue
+        if KUBEVIRT_CONSOLE_PLUGIN in pod.name and _is_console_plugin_bug_open():  # type: ignore[operator]
+            LOGGER.warning(f"Ignoring pod: {pod.name} due to CNV-82451 bug")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)